### ISSUE TITLE
Confused error message on unknown file

### ISSUE THREAD

**Issue by ai** [stylelint/stylelint#2251]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

UX bug

> Which version of stylelint are you using?

`7.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

```sh
stylelint file/doesnt/exists
```

> What did you expect to happen?

"File file/doesnt/exists doesn’t exists" error message.

> What actually happened (e.g. what warnings or errors you are getting)?

```sh
Error: Files glob patterns specified did not match any files
    at globby.then.filePaths (node_modules/stylelint/lib/standalone.js:71:33)
```

**Comment by jeddy3** [comment]
@ai Thanks for the request.

Is the issue here that the current error message doesn't distinguish between single files and globs? And you think having a specific error message for single files would be a better user experience?

**Comment by ai** [comment]
1. Of course. User didn’t enter a glob, but see some glob error. Also some users even don’t know what glob is.
2. Also we should write missed file name. If I saw what name was passed to Stylelint by lint-staged, debug could be faster. Maybe related also to issue #649.

**Comment by davidtheclark** [comment]
How about just `whether-you-entered does not match any files`?

```
file/doesnt/exist does not match any files

**/things/*.css does not match any files

file/one file/two and file/three do not match any files
```

**Comment by ai** [comment]
@davidtheclark nice idea :+1:

**Comment by davidtheclark** [comment]
Closed by #2328.
